### ISSUE TITLE
Screen Lock is asked to the User when user tries to login with Face Recognition ID, User is not directly login by face scan even though the user has allowed the Face login for the application.

### ISSUE THREAD

**Issue by iamrekhaverma** [expo/expo#11512]
### Summary of Issue
Screen Lock is asked to the User when a user tries to login with Face Recognition ID, User is not directly login by face scan even though the user has allowed the Face login for the application.

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
Note able to reproduce the issue on iOS devices having Face recognition feature

Repro on Devices:

Samsung Galaxy M30 S (OS: 9)

Samsung Galaxy M51 (OS:10)

Samsung Galaxy M21 (OS:10)

Samsung Galaxy F41 (OS:10)

Realme Pro3 (OS: 10)

### Reproducible Demo
 https://github.com/iamrekhaverma/LocalAuthExample
### Steps to Reproduce

Steps to Repro:

Add face recognition from Device setting [User must be ask to set Lock screen [PIN/Pattern/Password] from device setting, so set any of the lock ]]
Add following statement LocalAuthentication. authenticateAsync().

it will be asked for Lock screen [PIN/Pattern/Password]

User must have to unlock the Lock screen [PIN/Pattern/Password]

### Expected Behavior vs Actual Behavior
It should scan the user's face to authenticate user but it just fallback to screen lock.

**Comment by brentvatne** [comment]
does this return the expected values for you? https://docs.expo.io/versions/latest/sdk/local-authentication/#localauthenticationsupportedauthenticationtypesasync

it's not clear to me what your repository is meant to demonstrate. the code does not do what you have described here. it seems to just alert the `LocalAuthentication` object and then log the result of `hasHardwareAsync` -- which only tells you whether the device has fingerprint **or** face id and not whether enrolled or available

**Comment by iamrekhaverma** [comment]
> does this return the expected values for you? https://docs.expo.io/versions/latest/sdk/local-authentication/#localauthenticationsupportedauthenticationtypesasync
> 
> it's not clear to me what your repository is meant to demonstrate. the code does not do what you have described here. it seems to just alert the `LocalAuthentication` object and then log the result of `hasHardwareAsync` -- which only tells you whether the device has fingerprint **or** face id and not whether enrolled or available

@brentvatne I have updated the code just now. So when I click button it just fallback to screen lock although my phone supports facial recognition and it is enrolled in my device as a preffered biometric

**Comment by byCedric** [comment]
Please [see this discussion/PR](https://github.com/expo/expo/pull/11173), closing in favor of #10861

TL;DR; These Samsung devices aren't certified by Google to fully use the facial recognition features from Android. It's related to the hardware capabilities which are deemed "not capable enough". But, because we don't encrypt/decrypt key stores with this, we could add the non-certified and Samsung specific facial recognition system features. (cc @brentvatne)
